python/metatensor-operations/tests/block_from_array.py

```diff
@@ -50,3 +58,28 @@ def test_with_components():
     np.testing.assert_equal(
         block.properties.values, np.arange(array.shape[2]).reshape((-1, 1))
     )
+
+
+@pytest.mark.skipif(not HAS_TORCH, reason="requires torch")
+def test_torch_with_components():
```

Comment:
The bug here was that there were no test for the `block_from_array` for torch?
it has nothing to do with add, right?

Reply:
great